wireapp/kalium
# chore: add formatted logs for mls observability (WPB-6335)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

MLS was missing some logs format so we can monitor and create dashboards for observability

### Causes (Optional)

Difficult to track issues when this happens.

### Solutions

- Create an ext function to reuse and format logs in json format.
- Add logs to message sending and join via external commit.

### Testing

#### Test Coverage (Optional)


#### How to Test

On AR using this branch if you use the app, you should see logs there formatted.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
